plugins/org.jkiss.dbeaver.ext.postgresql.ui/src/org/jkiss/dbeaver/ext/postgresql/ui/PostgreConnectionPage.java

```diff
@@ -297,6 +301,24 @@ public void saveSettings(DBPDataSourceContainer dataSource) {
         super.saveSettings(dataSource);
     }
 
+    private static void applyGlobalProperties(DBPConnectionConfiguration connectionInfo) {
+        // Additional page can be not loaded at all, if this is the case just to be sure we need to apply global settings
+        String[] propertiesToCheck = {
+            PostgreConstants.PROP_SHOW_NON_DEFAULT_DB,
+            PostgreConstants.PROP_SHOW_UNAVAILABLE_DB,
+            PostgreConstants.PROP_SHOW_TEMPLATES_DB,
+            PostgreConstants.PROP_READ_ALL_DATA_TYPES,
+            PostgreConstants.PROP_READ_KEYS_WITH_COLUMNS,
+            PostgreConstants.PROP_SHOW_DATABASE_STATISTICS
+        };
```

Comment:
I don't like an idea of enumerating properties.

Reply:
The idea is that we don't need to set properties from additional page. Setting all properties looks really bad (we will push a lot of useless stuff in all connections and we won't be able to change defaults after it)